CHANGELOG.md

```diff
@@ -112,6 +112,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Enhance WebUI modals scrolling.
 
+- Success & error notifications in Schema editor.
+
```

Comment:
Don't add changes to the section 2.0.0, it's already released.

Reply:
ok